.github/workflows/tests.yaml

```diff
@@ -465,6 +465,15 @@ jobs:
             - name: Run Tests
               timeout-minutes: 65
               run: |
+                  scripts/run_in_build_env.sh \
```

Comment:
How about separating the two tests? Then we can give a title like `Run Discover Test` and `Run Pairing Test`